demos/backbone-requirejs/js/routers/mobileRouter.js

```diff
@@ -45,7 +45,10 @@ define([
         home: function() {
 
             // Programatically changes to the categories page
-            $.mobile.changePage( "#categories" , { reverse: false, changeHash: false } );
+            $( ":mobile-pagecontainer" ).pagecontainer( "change", "#categories", {
```

Comment:
We should avoid `:mobile-pagecontiner` its very very slow. You always know what your pagecontainer is currently. Either body or something else that you have set it to. So since in this ( and all other ) demos, it can be assumed to be body lets use that.